src/cookies.test.ts

```diff
@@ -36,11 +41,21 @@ describe('Cookies', () => {
     const iabConsentString = 'heL10W0rLd';
 
     beforeAll(() => {
+        global.Date = {
+            now: () => '12345',
+        };
         Object.defineProperty(document, 'domain', {
             value: 'www.theguardian.com',
         });
     });
 
+    afterAll(() => {
+        global.Date = OriginalDate;
+        expect(new Date().toString()).not.toMatch(
```

Comment:
This test doesn't make sense to me. We are not setting Date() to the beginning of time so it will always return true.